### PR TITLE
SUBMARINE-126. Reorganize the documentation of mini-submarine

### PR DESCRIPTION
**Pull request by yuanzac**
### What is this PR for?
Refactor the documentation of mini-submarine:

Merge update-submarine-verision.md with README
Fix the typo of run_submarine* scripts.

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/SUBMARINE-126

### How should this be tested?
* First time? Setup Travis CI as described on https://submarine.apache.org/contribution/contributions.html#continuous-integration
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? Yes
* Does this needs documentation? Yes

### CODE REVIEW

**Comment by yuanzac** [comment]
Sure, PR context is updated.

File: dev-support/mini-submarine/README.md [hunk 1]
```diff
@@ -69,40 +69,102 @@ hdfs dfs -ls /user
...
+1. Submarine package name error
+
+   Because the package name of submarine 0.3.0 or higher has been changed from `apache.hadoop.submarine` to `apache.submarine`, So you need to set the Runtime settings in the `/usr/local/hadoop/etc/hadoop/submarine.xml` file.
+
+   ```
+   <configuration>
+      <property>
+        <name>submarine.runtime.class</name>
+        <!--
+        <value>org.apache.hadoop.yarn.submarine.runtimes.tony.TonyRuntimeFactory</value>
```
Comment (jiwq) [inline dev-support/mini-submarine/README.md]:
```suggestion
        <value>org.apache.submarine.runtimes.tony.TonyRuntimeFactory</value>
```

Comment (yuanzac) [reply dev-support/mini-submarine/README.md]: Thanks, Wanqiang.
It makes sense. just updated it.

File: dev-support/mini-submarine/README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ docker push hadoopsubmarine/mini-submarine:0.2.0
 #### Run mini-submarine image
 
 ```
-docker run -it -h submarine-dev --net=bridge --privileged local/hadoop-docker:submarine /bin/bash
+docker run -it -h submarine-dev --net=bridge --privileged local/mini-submarine:0.2.0 /bin/bash
```
Comment (jiwq) [inline dev-support/mini-submarine/README.md]: If we pull image from remote, we'd better modify the instructions.

Comment (yuanzac) [reply dev-support/mini-submarine/README.md]: Agreed, let me update it.

**Comment by yuanzac** [comment]
To eliminate the potential code conflicts with next PR of SUBMARINE-97, let me merge it first.

**Pull request merged**